lib/FrameManager.js

```diff
@@ -130,6 +130,14 @@ class FrameManager extends EventEmitter {
     this._frames.delete(frame._id);
     this.emit(FrameManager.Events.FrameDetached, frame);
   }
+
+  /**
+   * @param {!Frame} frame
+   * @return {boolean}
+   */
+  isMainFrameLoadingFailed() {
```

Comment:
I don't see much difference between the names (and I like the former more since it resembles protocol event names..)